src/core/src/bootstrap/Constants.py

```diff
@@ -275,3 +275,20 @@ class EnvLayer(EnumBackport):
         PRIVILEGED_OP_REBOOT = PRIVILEGED_OP_MARKER + "Reboot_Exception"
         PRIVILEGED_OP_EXIT = PRIVILEGED_OP_MARKER + "Exit_"
 
+    # Package / Patch State Ordering Constants
```

Comment:
Expand comment to an additional line to describe why this is being done.

Reply:
Maybe even encapsulate that in the names. As an example: PackageClassificationOrderInStatusReporting   (if the variable is not directly used too many times).

Reply:
Will do both